chapters/negative_binomial_20250310.qmd

```diff
@@ -36,6 +38,152 @@ options(dplyr.summarise.inform = FALSE)
+
+
+\begin{split}
+
+P(Y = y |\lambda) = \frac{e^{-\lambda}\lambda^{y}}{y!}, \ \ x = 0,1,2,\ldots \\
+
+f(\lambda) = \frac{\beta^\alpha}{\Gamma(\alpha)}\lambda^{\alpha-1}e^{-\lambda\beta}, \lambda > 0 \\
+
+\end{split}
+
+
+### The Unconditional Probability $P(Y = y)$
+
+Now if we want the $P(Y = y)$, we marginalize out $\lambda$:
+
+\begin{split}
+
+P(Y = y) &= \int_{0}^{\infty}P(Y=y|\lambda)f(\lambda)d\lambda \\
+
+&= \int_{0}^{\infty}\frac{e^{-\lambda}\lambda^{y}}{y!}\frac{\beta^\alpha}{\Gamma(\alpha)}\lambda^{\alpha-1}e^{-\lambda\beta}d\lambda \\
+
+&= \frac{\beta^\alpha}{y!\Gamma(\alpha)}\int_{0}^{\infty}e^{-\lambda}\lambda^y\lambda^{\alpha-1}e^{\lambda\beta}d\lambda \\
+
+&= \frac{\beta^\alpha}{y!\Gamma(\alpha)}\int_{0}^{\infty}\lambda^{\alpha + y -1}e^{\lambda\beta-\lambda}d\lambda \\
+
+&= \frac{\beta^\alpha}{y!\Gamma(\alpha)}\int_{0}^{\infty}\lambda^{\alpha + y -1}e^{-\lambda(\beta+1)}d\lambda  \ \ & &\text{kernel of } Gamma(\alpha+y,\beta+1)\\
+
+&= \frac{\beta^\alpha}{y!\Gamma(\alpha)}\frac{\Gamma(\alpha+y)}{(\beta+1)^{\alpha+y}} \\
+
+&= \frac{\beta^\alpha\Gamma(\alpha+y)}{(\beta+1)^{\alpha}(\beta+1)^yy!\Gamma(\alpha)} \\
+
+&= \frac{\Gamma(\alpha+y)}{y!\Gamma(\alpha)}\left(\frac{\beta}{\beta+1}\right)^\alpha\left(\frac{1}{\beta+1}\right)^y & &\text{collect terms} \\
+
+&= \frac{(a + y - 1)!}{y!(a-1)!}\left(\frac{\beta}{\beta+1}\right)^\alpha\left(\frac{1}{\beta+1}\right)^y & &\text{convert gamma to factorial} \\
+
+&= \frac{(a + y - 1)!}{y!(a-1)!}\left(p\right)^\alpha\left(1-p\right)^y & & \text{let } p = \frac{\beta}{\beta+1} \\
+
+
+&= {{\alpha+y-1}\choose{y}}\left(p\right)^\alpha\left(1-p\right)^y \sim NB(\alpha, p)
+
+\end{split}
+
+:::{.callout}
+**The unconditional probability $P(Y=y)$ follows a negative binomial distribution, where, $\alpha$ is the number of successes, and $y$ is the number of failures.** 
+:::
+
+
+### How does this ensure $Var(Y) > E[Y]$?
+
+This adds additional variability to $Y$ via the **law of total variance**:
+
+\begin{split}
+Var[Y] &= E[Var(Y|\lambda)] + Var(E[Y|\lambda]) \\
+
+&= E[\lambda] + Var(\lambda) \\
+
+&= \frac{\alpha}{\beta} + \frac{\alpha}{\beta^2} \\
+
+&= \mu + \frac{\mu^2}{\alpha} \ \, \text{where } \mu = \alpha / \beta
+
+\end{split}
+
+Compare this variance $Var(Y) = E[\lambda] + Var(\lambda)$ to the original Poisson variable $Var(Y|\lambda) = \lambda$. We've inflated the variance of $Y$ by treating $\lambda$ as random.
+
+
+In this parametrization, the variance of $Y$ is quadratic in $\mu$ at a rate controlled by the **dispersion parameter** $\alpha$, which is also the number of "successes" in the negative binomial distribution.
+
+In terms of p:
+
+
+\begin{split}
+\mu &= \frac{\alpha}{\beta} \\
+
+&= \frac{\alpha}{\frac{p}{1-p}} \text{, } \ \ \beta = \frac{p}{1-p} \\
+
+&= \alpha\left(\frac{1-p}{p}\right)
+
+\end{split}
+
+Which is the typical form of the $E[X]$ for $X \sim NB(\alpha,p)$
+
+
+### Case Study: Negative Binomial Distribution in Modern Policy
```

Comment:
Leave this in the introduction. Everything between lines 52-152 should be moved down to an appendix.